projects/models.py

```diff
@@ -31,5 +31,18 @@ def __str__(self):
         return self.title
 
 
+class Contact_Form(models.Model):
```

Comment:
This is not a form but a model
you should rename it to something like
`class contact_details(models.Model):`